src/main/java/frc/robot/competition/CompetitionRobotContainer.java

```diff
@@ -144,7 +142,34 @@ private void configureBindings() {
                 m_driveController::getRightTriggerAxis,
                 RobotConstants.TRANSLATION_PID,
                 RobotConstants.ROTATION_PID,
-                RobotConstants.MAX_SPEED));
+                RobotConstants.MOTION_LIMITS.maxSpeed,
+                m_poseEstimator));
+    m_driveController
+        .a()
+        .or(m_driveController.b())
+        .or(m_driveController.x())
+        .or(m_driveController.y())
```

Comment:
Let's start by splitting these into distinct separate commands, each one attached to a specific angle. Then we can build up combining them

Reply:
I think it would be more annoying to have separate commands for different buttons or button combinations than the existing direction calculation, as it has been quite robust and accounts for 45 degree angles and opposite buttons cancelling